CMakeLists.txt

```diff
@@ -17,6 +17,17 @@ set(PROJECT_COMPILE_OPTIONS
   -Wno-unused-parameter
 )
 
+# Set compile options
```

Comment:
this is duplicate with the block above. I'll remove this when fast-forwarding to `ros2`